src/compiler/types.ts

```diff
@@ -373,8 +373,10 @@ namespace ts {
         // https://jsdoc.app/about-namepaths.html
         JSDocNamepathType,
         JSDocComment,

```

Comment:
this new name is 100% silly -- I would *like* to change the existing `JSDocComment` to `JSDoc` and `JSDocCommentComment` to `JSDocComment`, but I'm not sure how bad of a breaking change that is. Maybe CommentComment should just become CommentText. @DanielRosenwasser do you have opinions?

Reply:
CommentText seems good!

Reply:
![image](https://user-images.githubusercontent.com/972891/101848233-d1785200-3b09-11eb-8d88-b96a03182c64.png)

No GitHub, it should be "Update *comment text*"